docker/ds/Dockerfile

```diff
@@ -15,7 +15,7 @@ COPY bootstrap/ /var/tmp/bootstrap/
 COPY schema /var/tmp/schema
 
 # The default BASE_DN of the directory tree.
-ENV BASE_DN dc=data
+ENV BASE_DN dc=ou=identities
```

Comment:
Shouldn't this be "ou=identities" without the "dc"

Reply:
good catch. Fixed